1-exercises/B-highlight-words/script.js

```diff
@@ -1,5 +1,25 @@
 function highlightWords(paragraph, colours) {
-  // Write your code here...
+  const paraText = document.createElement("p");
+  const selectColor = document.createElement("select");
+
+  document.querySelector("#content").appendChild(paraText);
+  document.querySelector("#content").appendChild(selectColor);
+
+  ["Choose a Color", ...colours].forEach((color) => {
+    let optionColor = document.createElement("option");
+    optionColor.innerText = color[0].toUpperCase() + color.slice(1);
+    selectColor.appendChild(optionColor);
+  });
+
+  paragraph.split(" ").forEach((word) => {
+    const span = document.createElement("span");
+    span.innerText = word + " ";
+    paraText.appendChild(span);
+
+    span.addEventListener("click", () => {
+      span.style.backgroundColor = selectColor.value;
+    });
+  });
 }
 
```

Comment:
Well done Bahareh,
I would like to ask you to review your test, despite the fact the you have the correct resault but you need to do a small change.
Please run the test:
`npm test -- --testPathPattern B-highlight-word`
😊

Reply:
Thank you very much Mo , (i didnt know there is a test file for exercises). Updated